src/main/java/com/marklogic/mgmt/ManageClient.java

```diff
@@ -62,15 +62,12 @@ public void setManageConfig(ManageConfig config) {
 			    logger.info(format("Initializing separate connection to Manage API with user '%s' that should have the 'manage-admin' and 'security' roles", securityUsername));
 		    }
 
-		    RestConfig rc = new RestConfig(config.getHost(), config.getPort(), securityUsername, config.getSecurityPassword());
-		    rc.setScheme(config.getScheme());
-		    rc.setConfigureSimpleSsl(config.isConfigureSimpleSsl());
-		    rc.setHostnameVerifier(config.getHostnameVerifier());
-
+			RestConfig rc = new RestConfig(config);
+			// Override settings based on the 3 "security user"-specific properties known by ManageConfig
```

Comment:
@BillFarber What's going on here is that ml-app-deployer provides support for two sets of credentials, where the second is for a user with the "security" role. If securityUsername is set, then the "securityUserRestTemplate" is created based on securityUsername + securityPassword, and that's used for calls to the Manage API that require the "security" role. That's a powerful role, so the intent is to allow users to minimize how much is done with that user by using two sets of credentials.